### PR TITLE
Fix framework tests in iOS

### PR DESCRIPTION
**Pull request by PavelPunegov**
* Fix name in the path
* Don't copy static framework to the application bundle

### CODE REVIEW

File: backend.native/tests/build.gradle [hunk 1]
```diff
@@ -4108,6 +4108,7 @@ Task frameworkTest(String name, Closure<FrameworkTest> configurator) {
                         }
 
                         extraOpts fr.bitcode ? "-Xembed-bitcode" : "-Xembed-bitcode-marker"
+                        extraOpts fr.isStatic ? "-Xstatic-framework" : ""
```
Comment (SvyatoslavScherbina) [inline backend.native/tests/build.gradle:4111]: Is adding empty flag has the same effect as adding no flags?

Comment (SvyatoslavScherbina) [reply backend.native/tests/build.gradle:4111]: Well, adding empty argument is definitely different from not adding arguments at all.
The former might have unpredictable impact. For example:
```bash
$ konanc "" foo.kt
```
->
```text
error: source file or directory not found:
```

Comment (PavelPunegov) [reply backend.native/tests/build.gradle:4111]: Yes, I found out that Konan plugin runs konanc main with arguments in the `.lst` argfile

Comment (PavelPunegov) [reply backend.native/tests/build.gradle:4111]: Parser skips empty lines here checking for `isWhitespace`: https://github.com/JetBrains/kotlin/blob/master/compiler/cli/cli-common/src/org/jetbrains/kotlin/cli/common/arguments/preprocessCommandLineArguments.kt#L64
https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.text/is-whitespace.html

Comment (SvyatoslavScherbina) [reply backend.native/tests/build.gradle:4111]: My intention was to demonstrate that adding empty argument is error-prone and should be avoided in general, no matter what particular implementation does.

Please avoid adding `""`.

**Pull request merged**